code/controllers/subsystem/parallax.dm

```diff
@@ -1,34 +1,47 @@
 SUBSYSTEM_DEF(parallax)
 	name = "Parallax"
-	wait = 2
+	wait = 1
```

Comment:
It's even faster with `wait = 1`

Reply:
It also will get called 2x as frequently, although since its an SS_BACKGROUND system this will result in an increased CPU usage only when there is enough time to spare anyway. The idea for making it `wait = 2` was that the slight delay is barely noticable at slow speeds